framework/doc/content/source/partitioner/PetscExternalPartitioner.md

```diff
@@ -1,6 +1,8 @@
 # PetscExternalPartitioner
 
 Allow users to use several external partitioning packages (parmetis, chaco, ptscotch and party) via PETSc.
+Note that partitioning, just as meshing, requires a level of user insight to assure the domain is appropriately distributed. For example, edge cases where one seeks to have very few elements per process
+may misbehave with certain partitioners. To avert such situations, we switch ParMETIS to PTScotch in cases with less than 28 elements per process, and notify the user with a warning.
```

Comment:
I know it's hard to come up with a good answer to "at what point should we overrule the user and stop trusting Parmetis", but the answer in the docs is 28, the answer in the comment is 32, and the answer in the code is 20?  :-D

I don't see any other problems here, but I'll wait until after your planned squash/rebase/etc before submitting an official review.

Reply:
Good catch! With the going back and forth on the figs and doc I didn't get to make sure it's consistent everywhere. I would have noticed after civet runs and I give it another look. Annoying one can't run all the civet tests before pushing, the local tests aren't always enough.